sonar-python-plugin/src/test/java/org/sonar/plugins/python/SymbolVisitorTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.python;
+
+import java.io.File;
+import java.nio.charset.StandardCharsets;
+import java.util.Collection;
+import java.util.stream.Collectors;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.sonar.api.batch.fs.TextPointer;
+import org.sonar.api.batch.fs.TextRange;
+import org.sonar.api.batch.fs.internal.DefaultInputFile;
+import org.sonar.api.batch.fs.internal.DefaultTextPointer;
+import org.sonar.api.batch.fs.internal.TestInputFileBuilder;
+import org.sonar.api.batch.sensor.internal.SensorContextTester;
+import org.sonar.python.PythonVisitorContext;
+import org.sonar.python.TestPythonVisitorRunner;
+import org.sonar.python.api.tree.FileInput;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SymbolVisitorTest {
+
+  private static SensorContextTester context;
+  private static String componentKey;
+
+  @BeforeClass
+  public static void scanFile() {
+    String dir = "src/test/resources/org/sonar/plugins/python/squid-sensor";
```

Comment:
Why the dir variable ?  I think you can inline it and concatenate it to the file name.